app/components/oral_history/admin/combined_audio_derivatives_component.html.erb

```diff
@@ -39,9 +39,9 @@
       </div>
     <% end %>
 
-    <% if work_available_members? %>
+    <% if work_available_members?%>
```

Comment:
removing the space here is not an improvement but no big deal who cares just saying heh